keras_retinanet/backend/common.py

```diff
@@ -21,6 +21,18 @@
 
 
 def bbox_transform_inv(boxes, deltas, mean=None, std=None):
+    """ Applies deltas (usually regression results) to boxes (usually anchors).
```

Comment:
Maybe good to add the fact that it applies deltas and also undoes normalization.